Feat improve disabling read only documentation (#15783)
supabase/supabase: apps/docs/pages/guides/platform/database-size.mdx

You can manually override read-only mode to reduce disk size. To do this, run the following in the [SQL Editor](https://supabase.com/dashboard/project/_/sql):

To change the [transaction access mode](https://www.postgresql.org/docs/current/sql-set-transaction.html) for the current transaction:

```sql
SET TRANSACTION READ WRITE;
```

This allows you to delete data from within the session. After deleting data, you should run a vacuum to reclaim as much space as possible.
You can also change the default transaction access mode for all subsequent transactions of a session:

```sql
SET SESSION CHARACTERISTICS AS TRANSACTION READ WRITE;
```

> Ater deleting data, you should run a vacuum to reclaim as much space as possible.

Reviewer: ```suggestion
Ater deleting data, consider running `vacuum` to reclaim as much space as possible:
```
